packages/dds/tree/src/simple-tree/treeApi.ts

```diff
@@ -208,6 +209,36 @@ export const treeNodeApi: TreeNodeApi = {
 	},
 };
 
+/**
+ * Returns a schema for a value if the value is a {@link TreeNode} or a {@link TreeLeafValue}.
+ * Returns undefined for other values.
+ */
+function tryGetSchema<T>(value: T): undefined | TreeNodeSchema<string, NodeKind, unknown, T> {
+	type TOut = TreeNodeSchema<string, NodeKind, unknown, T>;
+	switch (typeof value) {
+		case "string":
+			return stringSchema as TOut;
+		case "number":
+			return numberSchema as TOut;
+		case "boolean":
+			return numberSchema as TOut;
```

Comment:
```suggestion
			return booleanSchema as TOut;
```

Reply:
Ya, tests should not be passing with that. I gotta add some tests.....

Reply:
While updating this I got confused by two files name treeApi.ts, so I cleaned up/deduplicated their tests, and renamed one of them to better match its exports.

Reply:
Thanks, that's good cleanup!